src/collectd.conf.pod

```diff
@@ -7129,6 +7129,12 @@ want to set B<metadata.broker.list> to your Kafka broker list.
 
 =back
 
+=head2 Plugin C<write_log>
```

Comment:
Plugins without any configure options aren't usually mentioned in the manpage (which is something that could be discussed). On the other hand, they are listed in the README, which isn't the case yet for this plugin.